sdk/contentsafety/ai-content-safety-rest/src/contentSafetyClient.ts

```diff
@@ -40,6 +33,10 @@ export default function createClient(
     loggingOptions: {
       logger: options.loggingOptions?.logger ?? logger.info,
     },
+    credentials: {
+      scopes: options.credentials?.scopes ?? ["https://cognitiveservices.azure.com/.default"],
+      apiKeyHeaderName: options.credentials?.apiKeyHeaderName ?? "Ocp-Apim-Subscription-Key",
```

Comment:
Just curious what would happen if we specify both of the scopes and api key header name ?